kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -246,7 +248,25 @@ public void put(Collection<SinkRecord> records) {
 
     for (SinkRecord record : records) {
       if (record.value() != null || config.getBoolean(BigQuerySinkConfig.DELETE_ENABLED_CONFIG)) {
-        PartitionedTableId table = getRecordTable(record);
+        PartitionedTableId table;
+        if(enableRetries) {
+          try {
```

Comment:
Wondering if it would make more sense to wrap whole of the for loop in try-catch as we can, in future, have exceptions arising from rest of the places as well ?

Reply:
I thought so yes. I missed it while doing the code. I have created the change as a new commit. Please review.

Reply:
For this, I am thinking we can generate rc version first and check on cloud before moving ahead with prod release